schema.json

```
@@ -1,329 +1,423 @@
+  "$schema": "https://specs.frictionlessdata.io/schemas/table-schema.json",
+  "name": "dispositif-aide",
+  "title": "Dispositifs d'aides",
+  "description": "Spécification du fichier d'échange relatif aux dispositifs d'aides.",
+  "keywords": [
+    "dispositifs",
+    "aide"
+  ],
+  "countryCode": "FR",
+  "homepage": "https://github.com/etalab/schema-dispositif-aide",
+  "path": "https://github.com/etalab/schema-dispositif-aide/raw/v0.1.0/schema.json",
+  "licenses": [
+    {
+      "title": "Etalab Licence Ouverte 2.0",
+      "name": "etalab-2.0",
+      "path": "https://spdx.org/licenses/etalab-2.0.html"
+    }
+  ],
+  "resources": [
+    {
+      "title": "Fichier valide (CSV)",
+      "name": "exemple-valide-csv",
+      "path": "https://github.com/etalab/schema-dispositif-aide/raw/v0.1.0/exemple-valide.csv"
+    }
+  ],
+  "sources": [],
+  "created": "2022-03-14T00:00:00Z",
+  "lastModified": "2025-05-28T00:00:00Z",
+  "version": "v0.1.0",
+  "contributors": [
+    {
+      "title": "Geoffrey Aldebert",
+      "email": "geoffrey.aldebert@data.gouv.fr",
+      "organisation": "Etalab",
+      "role": "author"
+    },
+    {
+      "title": "Sixte De Maupeou",
+      "email": "sixte.de-maupeou@data.gouv.fr",
+      "organisation": "Etalab",
+      "role": "contributor"
+    },
+    {
+      "title": "Jean-Baptiste Le Dévéhat",
+      "email": "jeanbaptiste.ledevehat.ext@ademe.fr",
+      "organisation": "ADEME",
+      "role": "contributor"
+    },
+    {
+      "title": "Cedric Merouani",
+      "email": "cedric.merouani@developpement-durable.gouv.fr",
+      "organisation": "CGDD",
+      "role": "contributor"
+    },
+    {
+      "title": "Marie Ramon-Dare",
+      "email": "marie.ramon-dare@developpement-durable.gouv.fr",
+      "organisation": "CGDD",
+      "role": "contributor"
+    },
+    {
+      "title": "Elise Marion",
+      "email": "elise.marion@developpement-durable.gouv.fr",
+      "organisation": "CGDD",
+      "role": "contributor"
+    },
+    {
+      "title": "Laurent Morice",
+      "email": "laurent.morice@ademe.fr",
+      "organisation": "Ademe",
+      "role": "contributor"
+    },
+    {
+      "title": "Thierry Verrechhia",
+      "email": "thierry.verrecchia@beta.gouv.fr",
+      "organisation": "beta.gouv",
+      "role": "contributor"
+    },
+    {
+      "title": "Benoit Delamea",
+      "email": "benoit.delamea@capgemini.com",
+      "organisation": "Cap Gemini",
+      "role": "contributor"
+    },
+    {
+      "title": "Thibaut Tezenas",
+      "email": "thibaut.tezenas@murmures.coop",
+      "organisation": "Murmures",
+      "role": "contributor"
+    }
+  ],
+  "fields": [
+    {
+      "name": "id_aide",
+      "description": "ID de l'aide. Identifiant local de l'aide du porteur (unique pour le porteur). Si vous ne disposez pas d'un identifiant, vous pouvez en générer un sur https://heidi.app.etalab.studio/",
+      "example": "65d5b6c7-102c-4440-ac3b-768f708edc0a",
+      "type": "string",
```

Si c'est bien le cas (cf https://specs.frictionlessdata.io//table-schema/#string)
```suggestion
      "type": "string",
      "format": "uuid",
```